src/Collector/BoolCollector.php

```diff
@@ -31,7 +31,7 @@ public function satisfy(
         }
 
         if (!$configuration['must'] && !$configuration['must_not']) {
-            throw new \InvalidArgumentException('bool collector must have a must or a must_not attribute');
+            throw new \InvalidArgumentException('"bool" collector must have a "must" or a "must_not" attribute.');
```

Comment:
was unrelated, but I think it's better to quotes those terms to be able to tell that they are special keywords and not just another word of the sentance